### PR TITLE
Include traits max value if possible

### PR DESCRIPTION
**Pull request by osdnk**
Fixes RNBW-####

## What changed (plus any additional context for devs)

I noticed that some trains are some value on the range and not including the max value doesn't make sense. 
E.g., see the screenshot:

## PoW (screenshots / screen recordings)
HAIR COLOR

before:
image

## Dev checklist for QA: what to test

- Go to Dame.eth
- Observe witches

### CODE REVIEW

File: src/components/Tag.js [hunk 1]
```diff
@@ -43,8 +44,8 @@ const OuterBorder = styled(Centered)`
   z-index: 2;
 `;
 
-const Text = styled(TextElement).attrs(({ theme: { colors } }) => ({
-  color: colors.whiteLabel,
+const Text = styled(TextElement).attrs(({ theme: { colors }, max }) => ({
+  color: max ? lightModeThemeColors.grey : colors.whiteLabel,
```
Comment (osdnk) [inline src/components/Tag.js:48]: I use `lightModeThemeColors`, because the layout of this screen is not changing with theme

**Review by skylarbarrera — APPROVED** [review]
this lgtm, post merging dev and CI passing

File: src/components/Tag.js [hunk 2]
```diff
@@ -60,7 +61,7 @@ const Title = styled(TextElement).attrs(({ color, theme: { colors } }) => ({
   margin-bottom: 1;
 `;
 
-const Tag = ({ color, disableMenu, slug, text, title, ...props }) => {
+const Tag = ({ color, disableMenu, slug, text, title, maxValue, ...props }) => {
```
Comment (jinchung) [inline src/components/Tag.js:64]: should the `Tag` component remain generic (not worry itself about a specific usage property such as `maxValue`?)

Instead should the `maxValue` logic be handled in `UniqueTokenAttributes` and passed along with the `text` value?

**Comment by jinchung** [comment]
this will need a rebase as CI is failing due to the audit dep

**Review by terrysahaidak — APPROVED** [review]
I don't like the maxValue prop of  theTag component but don't know how to make it better without significant refactor. So it's ok for now.

I would leave some comment about refactoring it in the future.

**Comment by christianbaroni** [comment]
@osdnk pushed a few small changes to the styling, think it's easier to interpret in an x/y format:

Screen Shot 2022-01-25 at 7 34 48 PM

**Pull request merged**